README.rst

```diff
@@ -76,10 +76,10 @@ Using PyAbel can be simple. The following Python code imports the PyAbel package
 
 	import abel
 	original     = abel.tools.analytical.sample_image()
-	forward_abel = abel.transform(original,     direction='forward', method='hansenlaw'  )['transform']
-	inverse_abel = abel.transform(forward_abel, direction='inverse', method='three_point')['transform']
+	forward_abel = abel.transform(original, direction='forward', method='hansenlaw'  ).transform
+	inverse_abel = abel.transform(forward_abel, direction='inverse', method='three_point').transform
 
-Note: the ``abel.transform()`` function returns a Python ``dict`` object, where the 2D Abel transform is accessed through the ``'transform'`` key.
+Note: the ``abel.transform()`` class returns a Python ``class`` object, where the 2D Abel transform is accessed through the ``.transform`` attribute.
```

Comment:
I'm out of my element here, but I think that the language is that "abel.transform() returns an object (an instance of the abel.Transform class)